test.sh

```diff
@@ -145,7 +145,7 @@ while getopts lueciosvgrnhp:f:-: OPT; do
     s | start-py )                   RUN+=("start") ;;
     v | gomod-vendor )               RUN+=("gomod-vendor") ;;
     g | generate )                   RUN+=("generate") ;;
-    r | rpm )                        RUN+=("rpm") ;;
+    package )                        RUN+=("package") ;;
```

Comment:
1. We offer short-opts for each long-opt, we should try to keep to the style:
```suggestion
    m | make-artifacts )             RUN+=("make-artifacts") ;;
```
2. Since `p` was already taken, we could _opt_ for `m` for `make-artifacts`. We can replace `r` (for rpm) on line `131` with `m` for `make-artifacts` like so:
```shell
while getopts lueciosvgmnhp:f:-: OPT; do
```